biolink-model.yaml

```diff
@@ -4484,6 +4719,38 @@ classes:
       - UMLSST:shro
       - prov:Agent
 
+  agent:
+    is_a: provider
```

Comment:
I would say that Agent is the broadest class - a person, organization, or software that acts on its own accord in contributing to things.  The notion of an Agent is not specific to a publication contributor.  

My earlier comments were suggesting that we define only a single high level Agent class, and not specific subtypes for things like Provider, Publisher, Editor, Author, etc.

Reply:
@mbrush Hmm... ok. I had usurped the existing (prior to my PR) "Provider" class for that purpose originally, but you are simply tagging designation of "Provider" as another specific Agent role.  I guess that sounds ok, so I'll iterate one more time on this, although I'll leave it @cmungall and @deepakunni3 to comment further on this afterwards.